src/Package/Impl/Shell/VsAppShell.cs

```diff
@@ -89,6 +99,16 @@ public sealed class VsAppShell : IApplicationShell, IMainThread, IIdleTimeServic
             }
         }
 
+        private static VsAppShell GetInstance() {
```

Comment:
So basically code can import it, but it is useless until package calls Initialize right? Maybe then at least set CompositionService and ExportProvider in GetInstance?